js/class_add_remove.js

```diff
@@ -91,6 +91,7 @@ class BookCollection {
     this.updateLocalStorage(true);
   }
 }
+
 /**       Initialize the class before calling its methods       */
 const bookCollection = new BookCollection();
```

Comment:
- When I try to add books, the style crashes at the bottom.
![Screen Shot 2021-07-09 at 05 07 16](https://user-images.githubusercontent.com/22258274/125063102-b0161f00-e074-11eb-95ed-d5be2c2de94a.png)

Reply:
Thanks for pointing it out.
I solved this issue by given to the element body in css the property '  grid-template-rows: 10% auto 10%;' instead of  '' grid-template-rows: 10% 80% 10%".
Now the size of the app will expand avoiding the elements to overlap.